Meadowlark-js/packages/metaed-plugin-edfi-meadowlark/src/enhancer/JsonSchemaEnhancer.ts

```diff
@@ -42,6 +42,10 @@ const descriptorSchema: SchemaRoot = {
   title: 'EdFi.Descriptor',
   description: 'An Ed-Fi Descriptor',
   properties: {
+    id: {
+      type: 'string',
+      description: 'The item documentUuid.',
+    },
```

Comment:
You should remove these. We don't include them because, per the [ODS/API documentation](https://api.ed-fi.org/v6.1/docs/swagger/index.html?urls.primaryName=Descriptors), including `id` in a body throws an error.